src/QsCompiler/CompilationManager/EditorSupport/EditorCompletion.cs

```diff
@@ -661,5 +684,21 @@ private static Position GetDelimiterPosition(FileContentManager file, CodeFragme
                 IsIncomplete = isIncomplete,
                 Items = items?.ToArray()
             };
+
+        /// <summary>
+        /// Returns a flattened enumerable of all items in the <see cref="QsTuple{Item}"/>.
+        /// </summary>
+        private static IEnumerable<T> FlattenQsTuple<T>(QsTuple<T> tuple)
```

Comment:
You can replace that with a call to SyntaxGenerator.ExtractItems (admittedly probably not the best place to put that routine...).